source/PluginDev/Assets/GooglePlayGames/Platforms/Android/Developers/JavaObjWrapper.cs

```diff
@@ -291,52 +319,59 @@ public T InvokeCall<T>(string name, string sig, params object[] args)
             IntPtr method = AndroidJNI.GetMethodID(RawClass, name, sig);
             jvalue[] jArgs = ConstructArgArray(args);
 
-            if (method == IntPtr.Zero)
+            try 
             {
-                Debug.LogError("Cannot get method for " + name);
-                throw new Exception("Cannot get method for " + name);
-            }
+                if (method == IntPtr.Zero)
```

Comment:
Could we have this check before line:
jvalue[] jArgs = ConstructArgArray(args);

Reply:
Yeh, nice point.